platform/windows/os_windows.cpp

```diff
@@ -2137,7 +2137,7 @@ OS::TimeZoneInfo OS_Windows::get_time_zone_info() const {
 		ret.name = info.StandardName;
 	}
 
-	ret.bias = info.Bias;
+	ret.bias = -info.Bias;
```

Comment:
Can you add a comment to justify that `-`?

Reply:
Sure, will do

Reply:
Comment added :)